### ISSUE TITLE
AAFC collection manager priorities for tools/functions in the data management software

### ISSUE THREAD

**Issue by heathercole** [AAFC-BICoE/dina-planning#130]
This is to document tools/functional needed in the data management software, as identified AAFC collection managers; this is a starting point only, and should not be considered complete. Managers have included indications of how important these features are for managing their collection's data.

This is assuming there is an underlying database, that contains all the data fields AAFC managers need, that they can import data into it, export data out, and that all fields can be queried. 

[Homework.1.-.CNC.Database.Requirements.docx](https://github.com/AAFC-BICoE/dina-planning/files/6031735/Homework.1.-.CNC.Database.Requirements.docx)
[DINA.Functionality.CPVC.-.Jan.2021.-.RR.edits.pptx](https://github.com/AAFC-BICoE/dina-planning/files/6031736/DINA.Functionality.CPVC.-.Jan.2021.-.RR.edits.pptx)
[Homework1_DAO_DAOM.xlsx](https://github.com/AAFC-BICoE/dina-planning/files/6031737/Homework1_DAO_DAOM.xlsx)
[DB.requirements.for.CCAMF.pdf](https://github.com/AAFC-BICoE/dina-planning/files/6031738/DB.requirements.for.CCAMF.pdf)
[CCFC_requirementsforDINAmodules_github.xlsx](https://github.com/AAFC-BICoE/dina-planning/files/6031739/CCFC_requirementsforDINAmodules_github.xlsx)
[CPVC_Database.notes.-.Print.Table.Relationship.docx](https://github.com/AAFC-BICoE/dina-planning/files/6031742/CPVC_Database.notes.-.Print.Table.Relationship.docx)

**Comment by heathercole** [comment]
from CCFC: I would like whatever we need to include our specimens in the predicted "extended specimen network", i believe to make that possible we would need unique DOIs for each of our accessions. the goal is described here: https://academic.oup.com/bioscience/article/70/1/23/5637849

**Comment by dshorthouse** [comment]
@heathercole Will leave these to you to make actionable requirements tickets for the devs when the relevant modules are in flight.

**Comment by heathercole** [comment]
@dshorthouse 
No. I cannot effectively translate these requirements into dev actions/tickets. I don't have any expertise or insight as to the DINA implementation of these requirements communicated by the collection managers.

**Comment by dshorthouse** [comment]
@heathercole If the intent of these is as a placeholder to solely jog the memories of managers, then I suppose it's fine that they reside here. It does however run counter to the expected use of brief, to-the-point tickets that can be closed once work is completed and confirmed. None of us presently have the cycles to look at these and convert into requirements tickets and so they cannot be treated as contracts for expected work.

**Comment by heathercole** [comment]
This (github) has been identified as the space for documenting communication and requirements. This is documenting the communication of these requirements from the collection managers to WP3. There isn't always the time available for collections to provide information on the same schedule it is requested, therefore, this information is being provided as a starting point to WP3.

**Comment by cgendreau** [comment]
This is a parallel stream and like David mentioned we can't afford that. GitHub tracks thing as Issues that are actionable and we have weekly meeting to talk about that so there is no need for parallel documents.

**Comment by dshorthouse** [comment]
@heather You've correctly identified that this is the place where requirements are document but you've misconstrued the format. We are on the path now for using structured tickets using a template to accelerate development by reducing any and all cognitive effort that results from out-of-bound, out-of-context task-switching. Here is the template for new requirements https://github.com/AAFC-BICoE/dina-planning/issues/new?assignees=&labels=type%3A+requirement&template=requirement.md&title=. We are all now expected to use this.

**Comment by heathercole** [comment]
I will bring this up at the next BioMob Lead's meeting, there needs to be a venue for Collections to communicate and document requirements outside of WP3 timelines. It cannot be collection manager's responsibility to translate their requirements into action items.

**Comment by dshorthouse** [comment]
If collection managers do not have the time or capacity to do this then we best find someone to act as a liaison to collection managers.

**Comment by heathercole** [comment]
It would definitely make sense for someone with DINA system knowledge to support converting the requirements into dev. actions.

**Comment by ssbilkhu** [comment]
> 
> 
> It would definitely make sense for someone with DINA system knowledge to support converting the requirements into dev. actions.

I will get @TravisGobeil 's help to review the documents and create the any actionable developer tickets.
